### PR TITLE
ENH: use env variables for docker-compose

### PR DESCRIPTION
**Pull request by mvdoc**
With this PR all paths can be specified in the `.env` file, reducing the amount of data that needs to be copied when building the docker image. Everything can just be mounted.

Also the containers are set to restart always, in case something funky happens.

### CODE REVIEW

**Comment by eickenberg** [comment]
cool
uh but can't review this as I have no clue what's going on :)

**Comment by eickenberg** [comment]
test

**Comment by eickenberg** [comment]
test2

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PYCORTEX_STORE=./
+EVENT_DEVICE=/dev/input/event3
+PIPELINE_PATH=./realtimefmri/pipelines
+TEST_DATASET_PATH=./
```
[the remark this answers is not in the document]
Comment (r-b-g-b) [reply .env]: Agreed we should avoid commit. Disagree that it needs to be an env variable. I'll let you make the final decision though

**Pull request merged**